Makefile

```diff
@@ -51,7 +51,7 @@ install: ## install from source (runs setup.py)
 
 .PHONY: test
 test: dsp-stack ## run all tests
-	pytest test/
+	-pytest test/
```

Comment:
If the tests fail, `pytest test` has a non-zero exit status, so that `make` exits, too. But I want that `make` ignores the exit status and continues with the next command. See https://www.gnu.org/software/make/manual/make.html#Errors

Reply:
Good point. Both of the modified targets are intended for local usage, which I have now added in the comment line. And they are really not used by the CI.